ad-first-campaign.py

```diff
@@ -0,0 +1,66 @@
+def main():
```

Comment:
as discussed, let's put it into another branch :)

Reply:
Yes